install/helm/gloo/templates/6.5-gateway-certgen-job.yaml

```
@@ -69,6 +72,6 @@ spec:
 {{/* Render template with yaml overrides */}}
 {{- $kubeResourceOverride := dict -}}
 {{- if .Values.gateway.certGenJob -}}
-{{- $kubeResourceOverride = .Values.gateway.certGenJob.kubeResourceOverride -}}
+{{- $kubeResourceOverride = .Values.gateway.certGenJob.ValidationWebhookKubeResourceOverride -}}
```

this change should be reverted
```suggestion
{{- $kubeResourceOverride = .Values.gateway.certGenJob.kubeResourceOverride -}}
```